server/src/main/java/io/druid/server/coordination/ZkCoordinator.java

```diff
@@ -187,6 +191,8 @@ public void addSegments(Iterable<DataSegment> segments, final DataSegmentChangeC
 
       final List<ListenableFuture> segmentLoading = Lists.newArrayList();
 
+      final int numSegments = segments.size();
+      final AtomicLong counter = new AtomicLong(0);
```

Comment:
should we change it then? someone might assume there's a reason for it to be a long otherwise.